src/MOI_wrapper/results.jl

```diff
@@ -3,16 +3,16 @@ function MOI.get(model::Optimizer, ::MOI.TerminationStatus)
 end
 
 function MOI.get(model::Optimizer, ::MOI.ObjectiveValue)
-    if model.status == MOI.OPTIMIZE_NOT_CALLED
-        @error "optimize! not called"
-    end
+    # if model.status == MOI.OPTIMIZE_NOT_CALLED
+    #     @error "optimize! not called"
+    # end
```

Comment:
What is the reason for this? Isn't that the normal way? Additionally this is still in use for getting the value of a variable.

Reply:
woops yes sorry, I tried to remove the error throwing to make Zygote work (in the end, there were too many of them also in MOI), will revert that